kits/multisig/test/multisig.js

```diff
@@ -238,6 +238,32 @@ contract('Multisig Kit', accounts => {
                 assert.equal(state[7], 0, 'nay vote should have been removed')
             })
 
+            it('throws when non-holder tries to create a vote directly', async () => {
+                const action = { to: executionTarget.address, calldata: executionTarget.contract.execute.getData() }
+                script = encodeCallScript([action])
+                try {
+                    await voting.newVote(script, 'metadata', { from: nonHolder })
+                } catch (err) {
+                    assert.equal(err.receipt.status, 0, "It should have thrown")
+                    return
+                }
+                assert.isFalse(true, "It should have thrown")
```

Comment:
Might be nice just to use the `assertRevert` helpers we use in `aragon-apps` (it's published as [`@aragon/test-helpers`](https://www.npmjs.com/package/@aragon/test-helpers))

Reply:
If I recall correctly, that was not working with the geth node in our docker image.